jena-base/src/main/java/org/apache/jena/atlas/lib/CacheFactory.java

```diff
@@ -43,7 +43,7 @@
 
     /** One slot cache */
     public static <Key, Value> Cache<Key, Value> createOneSlotCache() {
-        return new Cache1<>() ;
+        return new CacheGuava<>(1) ;
     }
```

Comment:
CacheGuava is a bit heavy for a one-slot cache.  While Cahce1 may not be used currently, if it were, then keeping it as simple, small implementation is better. Proposal: Keep Cache1 as is.